content/en/docs/guides/custom-load-balancing.md

```diff
@@ -81,8 +81,9 @@ For more details, please see gRPC [proposal A52].
 | Language | Example        | Notes                            |
 |----------|----------------|----------------------------------|
 | Java     | [Java example] |                                  |
-| Go       |                | Example and xDS support upcoming |
-| C++      |                | Not yet supported                |
+| Go       | [Go example]   | Example and xDS support upcoming |
```

Comment:
You removed "Not yet supported" from C++ and left "Example and xDS support upcoming" for Go.